docs/pages/blog/premium-plan-release.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Premium passengers, please proceed to the boarding gate 🚀
+description: Introducing the MUI X Premium plan, and a new licensing model
+date: 2022-05-12T00:00:00.000Z
+authors: ['josefreitas', 'alexfauquette']
+tags: ['MUI X', 'News']
+---
+
+We’re happy to announce that the Premium plan is [finally out](https://mui.com/pricing/)!
+With it, MUI X officially steps up to the next level, supporting the most advanced use cases for UI components.
+
+The new Premium features are intended to support end-users with the most sophisticated tools to navigate and draw insights from vast amounts of data.
+
+## Row grouping
+
+We started with [row grouping](https://mui.com/x/react-data-grid/row-grouping/), which enables users to group data based on repeating values in the grid.
+For example, users can now group orders by buyers, movies by directors, or cities by the governing parties.
+
+These kinds of use cases would usually be dealt with using a different query on the database, perhaps requiring a new service end-point, maybe even a new front-end, with all the UX challenges this would entail.
```

Comment:
```suggestion
These kinds of use cases would usually be dealt with using a new query on the database, perhaps requiring a new service end-point, maybe even a new front-end, with all the UX challenges this would entail.
```